Sources/KituraContracts/CodableQuery/QueryDecoder.swift

```diff
@@ -29,6 +29,14 @@ import LoggerAPI
      return
  }
  ````
+ 
+ ### Decoding The Empty String: ###
+ HTML forms send the empty string ("") in query parameters when a field is empty (i.e. &key1=&key2=).
```

Comment:
I don't like the opening sentence.  We could interpret the absence of a value as an empty string, but I would prefer something like:

> ### Decoding empty values
> When an HTML form is sent with an empty or unchecked field, the corresponding key/value pair is sent with an empty value (i.e. `&key1=&key2=`).
> The corresponding mapping to Swift types performed by `QueryDecoder` is as follows:
(etc)

Reply:
(also, back-ticks around code elements)